docker/everest-playground-opensuse-tumbleweed/Dockerfile

```diff
@@ -53,4 +53,7 @@ RUN ssh-keyscan github.com > ~/.ssh/known_hosts
 ENV PATH="/home/$USERNAME/.local/bin:$PATH"
 ENV CPM_SOURCE_CACHE="/home/$USERNAME/cpm_source_cache"
 
+RUN echo "echo \"To initialize the EVerest workspace please run the following command:\"" >> /home/${USERNAME}/.bashrc
+RUN echo "echo \". init.sh\"" >> /home/${USERNAME}/.bashrc
+
```

Comment:
This could be put into a separate file when using $HOME, furthermore the  settings.xml for caching maven repos could be supplied as an additional file

Reply:
Absolutely, this could then also be setup in the devcontainer since that's the primary use of this message anyway.
We could also think about restructuring the `init.sh` so it can be invoked automatically during start of the devcontainer.